fasthx/main.py

```diff
@@ -17,52 +19,54 @@ def _append_to_signature(
     func: Callable[_P, _T],
     *params: inspect.Parameter,
 ) -> Callable[_P, _T]:
-    """
-    Appends the given parameters to the *end* of signature of the given function.
+    """Appends the given parameters to the *end* of signature of the given function.
```

Comment:
Please do not reformat the docstring. I understand that this is probably the most common style, but starting multiline docstrings on a newline is much more readable. Also, please don't add horizontal lines after Notes, Sections and Returns.

Reply:
Understand. I only changed it because VS Code was going bananas, the entire files was covered in red and yellow. Most of these docstring changes are just so that I can actually read the code. I think this is because I didn't load up your ruff configs, more below.